Kconfig.zephyr

```diff
@@ -324,6 +324,15 @@ config LINKER_USE_RELAX
 	  Disabling it can reduce performance, as the linker is no longer able to substiture long /
 	  in-effective jump calls to shorter / more effective instructions.
 
+config LINKER_ERROR_RWX_SEGMENTS
+	bool "Error on RWX output segments"
+	default n
```

Comment:
`default n` is redundant in this case.

```suggestion
```

Reply:
fixed (and left un-rebased, in the hope of making review easier)